### PR TITLE
DS-330

### PR DESCRIPTION
**Pull request by mozaki1**

### CODE REVIEW

**Comment by mozaki1** [comment]
Wagtail upgrade to 2.15

**Review by jacobtoppm — CHANGES_REQUESTED** [review]
Thanks @mozaki1! It looks like you've got some Poetry conflicts, and the CI currently isn't running against your PR - would you be able to rebase it on main, fix any conflicts, and push again so we can see if it passes CI?

File: etna/records/tests/test_blocks.py [hunk 1]
```diff
@@ -266,18 +268,22 @@ def test_add_featured_records(self):
...
-        self.assertEqual(len(responses.calls), 2)
+        self.assertEqual(
+            self.insights_page.body[0].value["introduction"],
+            "This is some text"
+        )
+        self.assertEqual(
+            self.insights_page.body[0].value["records"][0].iaid,
+            "C123456"
+        )
+        self.assertEqual(len(responses.calls), 3)
```
Comment (jacobtoppm) [inline etna/records/tests/test_blocks.py:286]: Why is there an extra API call now? Might be worth investigating

File: pyproject.toml [hunk 2]
```diff
@@ -8,10 +8,10 @@ authors = ["Dan Bentley <dan@numiko.com>"]
 python = "^3.8"
 gunicorn = "^20.1.0"
 elasticsearch = "^7.12.0"
-wagtail = "^2.12.3"
+wagtail = "2.15"
```
Comment (jacobtoppm) [inline pyproject.toml:11]: 2.15.1 is now released, which contains some bug fixes. Probably worth switching to that - it should be very straightforward

**Review by jacobtoppm — CHANGES_REQUESTED** [review]
Generally looks great! Just one thing to check and a quick update

File: etna/records/tests/test_blocks.py (hunk 1 above)
Comment (mozaki1) [reply etna/records/tests/test_blocks.py:286]: The call for self.insights_page.body[0].value["record"].iaid , calls FetchQuery(self.model).get_multiple(iaid=iaid) , which is another KongClient fetch.

**Review by jacobtoppm — APPROVED** [review]
Nice work! I'd suggest squashing the commits when we merge this in, but otherwise all looks good to me

**Pull request merged**